build.psm1

```diff
@@ -170,7 +170,8 @@ function Test-Win10SDK {
     #
     # A slightly more robust check is for the mc.exe binary within that directory.
     # It is only present if the SDK is installed.
-    return (Test-Path "${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64\mc.exe")
+    return (Test-Path "${env:ProgramFiles(x86)}\Windows Kits\10\bin\x64\mc.exe") -or
+           (Test-Path "${env:ProgramFiles(x86)}\Windows Kits\10\bin\10.0.*.0\x64\mc.exe")
```

Comment:
I see 5 folder on my computer and I don't know path default...
Why we test full path here and use env path search in line 236?
Maybe test `Get-Command -Application mc.exe`?

Reply:
>Why we test full path here and use env path search in line 236?

Line 236? You mean this?:
https://github.com/PowerShell/PowerShell/blob/c43ae6a110ec1c3555276d14e7b7b9bab3fa2487/build.psm1#L236

>Maybe test `Get-Command -Application mc.exe`?

`mc.exe` is not in `PATH` by default. It is `vcvarsall.bat` responsibility to locate installed SDK and add it to path:
https://github.com/PowerShell/PowerShell/blob/c43ae6a110ec1c3555276d14e7b7b9bab3fa2487/build.psm1#L265

Reply:
Sorry the line I meant
https://github.com/PetSerAl/PowerShell/blob/8bee005699178008ca78efa306b4d196c53c0a54/build.psm1#L266

On my computer the exe is in PATH
```
where.exe mc.exe
C:\Program Files (x86)\Microsoft SDKs\Windows\v7.1A\Bin\MC.Exe
```
Perhaps new installer doesn't add this in PATH.

Reply:
For me it is not in `PATH`, unless you call `vcvarsall.bat` in initialize developer environment:
```cmd
C:\>where.exe mc.exe
INFO: Could not find files for the given pattern(s).

C:\>"\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvarsall.bat" x64
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.0
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'

C:\>where.exe mc.exe
C:\Program Files (x86)\Windows Kits\10\bin\10.0.17134.0\x64\mc.exe
```

Reply:
Therefore, we can not test the existence of mc.exe before launching `vcvarsall.bat`, is it?

Reply:
As far as I understand, yes. But I am not the one who implement original check, so I can not state exact reasons, why it was implemented this way.